stock_orderpoint_automatic_creation/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
this line [is not needed anymore in python 3](https://stackoverflow.com/questions/37048761/coding-utf-8-on-python3/37048998#37048998)